modules/apps/remote-app/remote-app-admin-web/src/main/resources/content/Language.properties

```diff
@@ -1,5 +1,7 @@
 javax.portlet.title.com_liferay_remote_app_admin_web_portlet_RemoteAppAdminPortlet=Remote Apps
+js-portlet=JavaScript Portlet
```

Comment:
Are we sure we want to use the word "Portlet" here? The usage of "Widget" is already predominant in Portal.

It's interesting since you yourself mentioned in the steps that you should: `[...] add a Vanilla Counter **widget** to the page` and `Widget` is the section under which you can find it.

Reply:
Probably. Since this has not been defined formally (PM side) I used the technical term without thinking too much about it.

But if we want these artifacts be like the ones deployed by the JS Toolkit it would be wise to name them the same, as you say, so I'll change it :+1: .